src/resolvers/podcastResolvers.ts

```diff
@@ -18,30 +23,42 @@ export const Query = {
     _: any,
     { id }: QueryToPodcastArgs,
     context: Context,
-  ): Promise<GQLAudio> {
+  ): Promise<IAudioMetaInformation> {
     return fetchPodcast(context, id);
   },
   async podcastSearch(
     _: any,
     { pageSize, page }: QueryToPodcastSearchArgs,
     context: Context,
-  ): Promise<GQLAudioSearch> {
+  ): Promise<IAudioSummarySearchResult> {
     return fetchPodcastsPage(context, pageSize, page);
   },
   async podcastSeries(
     _: any,
     { id }: QueryToPodcastSeriesArgs,
     context: Context,
-  ): Promise<GQLPodcastSeries> {
+  ): Promise<IAudioMetaInformation> {
     return fetchPodcastSeries(context, id);
   },
   async podcastSeriesSearch(
     _: any,
     { pageSize, page }: QueryToPodcastSeriesSearchArgs,
     context: Context,
-  ): Promise<GQLPodcastSeriesSearch> {
+  ): Promise<IAudioSummarySearchResult> {
     return fetchPodcastSeriesPage(context, pageSize, page);
   },
 };
 
-export const resolvers = {};
+export const resolvers = {
+  AudioSearch: {
+    async results(
+      searchResult: IAudioSummarySearchResult,
+      _: any,
+      context: Context,
+    ) {
+      return await Promise.all(
+        searchResult.results.map(audio => fetchPodcast(context, audio.id)),
```

Comment:
Som eg nevnte i går er det kanskje overflødig å gjøre dette for alle podcaster når du søker på serier.

Reply:
Skal jeg bare slette den inntil videre?

Reply:
Ja, tenker det. Men du treng vel resolveren for episodes når du henter en serie. Episodene listes jo opp på seriesida og treng den ekstra informasjonen.

Reply:
AudioSearch-resolveren trigges vel bare av PodcastSearch, som ikke tas i bruk for øyeblikket.

Reply:
Eg meinte en tilsvarende resolver som utvider episodene med ekstra informasjon. Denne vil måtte benytte fetchPodcast.